lgsm/functions/info_glibc.sh

```diff
@@ -131,6 +131,9 @@ elif [ "${engine}" == "refractor" ]; then
 elif [ "${gamename}" == "Wolfenstein: Enemy Territory" ]; then
 	glibcrequired="2.2.4"
 	glibcfix="no"
+elif [ "${gamename}" == "Multi Theft Auto" ]; then
+	glibcrequired="NOT REQUIRED"
```

Comment:
Please double check and confirm this for me using "detect-glibc" option in LinuxGSM.